.drone.env

```diff
@@ -1,4 +1,4 @@
 # The test runner source for API tests
-APITESTS_COMMITID=fc686d8cd827fe7b2bea55894c781fb72ad24865
+APITESTS_COMMITID=3cdc3bc48599af7f598f58e8dd87e910a2fca3b6
```

Comment:
@2403905 this change can be removed - it is reporting a git conflict. 
The commit id was updated late last week in PR #3984